site/src/loci/Table.jsx

```diff
@@ -53,7 +53,7 @@ const cols = [
     name: "Description",
   },
   {
-    key: "position_hg38",
+    key: "position_base0_hg38",
```

Comment:
I can't figure out how to also access position_base1_hg38 here to correctly create the link.

Reply:
`key` is the field to consider as the "main" value for the cell. That's what the first argument in the `render` function below becomes. But you can also access any other field on the row as the second argument. See my comment below.